deploy/helm/sumologic/values.yaml

```diff
@@ -4407,6 +4407,122 @@ metadata:
       ## To use maxUnavailable, set minAvailable to null and uncomment the below:
       # maxUnavailable: 1
 
+## Configure optional OpenTelemetry Gateway in Agent mode
+otelgateway:
+  enabled: false
+  deployment:
+    nodeSelector: {}
+    tolerations: []
+    resources:
+      limits:
+        memory: 2Gi
+        cpu: 1000m
+      requests:
+        memory: 196Mi
+        cpu: 50m
+    ## Add custom labels only to otelagent daemonset.
+    podLabels: {}
+    ## Add custom annotations only to otelagent daemonset.
+    podAnnotations: {}
+    image:
+      repository: "public.ecr.aws/sumologic/opentelemetry-collector"
+      tag: "0.38.1-sumo"
+      pullPolicy: IfNotPresent
```

Comment:
How about we already try to use Sumo OT distro here? Less migration headaches later.

WDYT: @SumoLogic/open-source-collection-team @pmaciolek ?

Reply:
Only for the gateway?